crates/storage_models/src/query/payment_attempt.rs

```diff
@@ -92,38 +79,6 @@ impl PaymentAttempt {
         .await
     }
 
-    pub async fn find_last_successful_attempt_by_payment_id_merchant_id(
```

Comment:
we need this back, we can't use active attempt id in payment_intent to make a refund for a particular attempt.